### ISSUE TITLE
TASK: Agree with Flexion on GitFlow

### ISSUE THREAD

**Issue by vickimcfadden** [ustaxcourt/ef-cms#480]
Purpose: 

- Enable changes to get to the Court’s environments more quickly.
  - Enables the Court to demo and train against the Court’s data sets. 
  - Enables Flexion freedom to adapt and change their environments and data sets as desired for testing. 
- Enable the development team to look their best — accepts only work that meets acceptance criteria.
  - In-progress code slows down review process, leading to back-and-forth conversations about code changes. 
  - Unfinished changes reduces the quality of the code base used in production, leading to stability concerns (and Sleepless Mike). 

Vision: 

- The Court can review and accept changes from Flexion in **two business hours**.

What we’re trying to get there:  

- Adjusting guidance on pull request generation to allow reviewing changes separated by completed Bug / Feature, to enable streamlined review. 
- Tweaking the pull request review process on the Court side to streamline the code review checklist such that they can be completed in two business hours or less, not including back-and-forth discussion of changes.

New Acceptance Criteria for Flexion:
- [ ] Court is able to SEE all of the changes for a Bug / Feature comprehensively, separate from other changes. 
- [ ] PRs to the Court’s repository link to diffs of changes separated by Bug / Feature.
- [ ] Court only receives code that has been QAed on a Flexion environment with Full Data Set
- [ ] Changes associated with a critical bug or hotfix can be QAed on a Flexion environment with Full Data Set and suggested to the Court without requiring other Bugs / Features to be completed first. 

What’s not changing: 

- Jessica/Mike/Julia/Andrew/Mark/"The Court" still would like to be involved as soon as possible to collaborate on solutions as appropriate.  

Tasks:
- [x] Define Goals. 
- [x] Meet with Flexion to discuss the problems we're trying to solve. 
- [x] Develop AC as necessary. 
- [x] Meet with Flexion again
- [x] We got plan!

**Comment by mmarcotte** [comment]
Plan:
* After #475 is complete, a new PR from Flexion is to be cut every two days.
* During the Blackout Window, Flexion devises plan for how to ensure tested and completed work is delivered.
* Court requests improved documentation in PRs detailing completed work ***and*** in-progress work https://github.com/ustaxcourt/ef-cms/pull/485
